packages/host/app/components/editor/directory.gts

```diff
@@ -112,7 +111,7 @@ export default class Directory extends Component<Args> {
       .directory :deep(.icon) {
         width: var(--icon-length);
         height: var(--icon-length);
-        margin-right: var(--icon-margin);
+        margin-right: math.div(var(--icon-margin), 2);
```

Comment:
This is coming out to 0, I think because `math.div` is a Sass thing. But I guess it can just be removed?

<img width="227" alt="Boxel 2023-10-31 13-17-52" src="https://github.com/cardstack/boxel/assets/43280/35a68df0-5112-44b2-be24-e97f06f7644e">

Reply:
lol, whoops. I'll remove it